internal/restic/node_xattr_bsd.go

```diff
@@ -0,0 +1,7 @@
+// +build freebsd netbsd openbsd
+
+package restic
+
+func isXattrNoData(err error) bool {
+	return false //no such error
```

Comment:
NetBSD has ENODATA, so the comment is incorrect. You can check this by compiling

```
package foo

import "syscall"

var _ = syscall.ENODATA
```

with GOOS=netbsd, etc.

A better comment would be "xattr should never return ENODATA."

Reply:
updated